tools/project_manager/edit_common.inc

```diff
@@ -48,37 +53,35 @@ function language_list($language)
         $sec_language = '';
     }
 
-    $array_list = $GLOBALS['lang_list'];
+    echo _("Primary"), ": <select name='pri_language' required>\n";
+    maybe_echo_placeholder_option($pri_language);
+    echo_language_options($pri_language);
+    echo "</select>\n";
```

Comment:
Can you please add a `<br>` here to put these each on their own line?

Reply:
I've done this now.

Reply:
Looks good to me.